### PR TITLE
Only Create Fargate Profile On Deployed Fargate Task

### PR DESCRIPTION
**Pull request by sethAmazon**

### CODE REVIEW

File: terraform/eks/main.tf [hunk 1]
```diff
@@ -160,9 +161,10 @@ resource "kubernetes_service_account" "aoc-role" {
 }
 
 resource "kubernetes_service_account" "aoc-fargate-role" {
+  count = 1
```
Comment (jefchien) [inline terraform/eks/main.tf]: Shouldn't this also only be created on fargate deployments? It seems fargate specific.

Comment (jefchien) [reply terraform/eks/main.tf]: Should've refreshed.

**Comment by jefchien** [comment]
Have you tested this?

**Review by khanhntd — APPROVED** [review]
LGTM.

**Review by jefchien — APPROVED** [review]

**Comment by sethAmazon** [comment]
Yes I was able to bring up terraform in multiple repo. I was not able to run multiple validators since the validator will recreate when run on the same docker network. Our ci runs on multiple docker networks for containers will not be re created.

**Pull request merged**